### DISCUSSION TITLE
lineage-19.1 sync error

### DISCUSSION THREAD

**Discussion by nhglong** [ROM-builders/temporary#12753] [Q&A]
error: Cannot fetch LineageOS/android_device_generic_common from https://github.com/LineageOS/android_device_generic_common
error: Cannot fetch LineageOS/android_frameworks_base from https://github.com/LineageOS/android_frameworks_base
Updating files:  30% (18774/61881)
Updating files:  31% (19184/61881)
Updating files:  32% (19802/61881)
Updating files:  33% (20421/61881)
error: Cannot checkout LineageOS/android_frameworks_base: ManifestInvalidRevisionError: revision refs/heads/lineage-19.1 in LineageOS/android_frameworks_base not found

@Apon77 any solution to fix ?

**Comment by Apon77** [comment]
remove full dir

**Reply by nhglong** [reply]
problem solved
